packages/web-wallet/src/types/credential/index.ts

```diff
@@ -96,20 +99,22 @@ export class CredentialTableItem {
       ...(credentialSummary?.branding?.background?.color && {backgroundColor: credentialSummary?.branding?.background?.color}),
     }
 
+    const vc = JSON.parse(credential.uniformDocument ?? credential.rawDocument ) as IVerifiableCredential
+    const vcType = (Array.isArray(vc.type) ? (vc.type as string[]).find(type => type !== 'VerifiableCredential')
```

Comment:
I think it's best to add this bit to the credential-mapper in the credential-branding lib. that way we can use the same logic in the mobile

Reply:
SDK-24 is already approved & merged Added a TODO and als moved the parsing to the top of the function because it was parsing twice